5.0/Dockerfile

```diff
@@ -12,12 +12,17 @@ RUN set -x \
 	&& chmod +x /usr/local/bin/gosu \
 	&& gosu nobody true
 
-# https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-repositories.html
-# https://packages.elasticsearch.org/GPG-KEY-elasticsearch
+# https://www.elastic.co/guide/en/elasticsearch/reference/5.0/deb.html
+# https://artifacts.elastic.co/GPG-KEY-elasticsearch
 RUN apt-key adv --keyserver ha.pool.sks-keyservers.net --recv-keys 46095ACC8548582C1A2699A9D27D666CD88E42B4
 
-ENV ELASTICSEARCH_VERSION 5.0.0-alpha5
-ENV ELASTICSEARCH_REPO_BASE http://packages.elasticsearch.org/elasticsearch/5.x/debian
+# add support for method of driver https
+RUN apt-get update \
+    && apt-get install -y --no-install-recommends apt-transport-https \
```

Comment:
Should we perhaps move this `apt-get install` section above the `apt-key adv`, since this line is less likely to change?  Not that the key should change anytime soon.